GirafIntegrationTest/tests/test_department_controller.py

```diff
@@ -187,9 +195,10 @@ def test_department_can_get_nonexistent_department_should_fail(self):
 
         Endpoint: GET:/v1/Department/{id}
         """
-        response = get(f'{BASE_URL}v1/Department/-2').json()
-        self.assertFalse(response['success'])
-        self.assertEqual(response['errorKey'], 'UserNotFound')
+        response = get(f'{BASE_URL}v1/Department/-2')
+        response_body = response.json()
+        self.assertEqual(response.status_code, HTTPStatus.NOT_FOUND)
+        self.assertEqual(response_body['errorKey'], 'UserNotFound')
```

Comment:
Minor thing, but when you are only asserting on one field of the response in the test, you can make it more pythonic with

```py
self.assertEqual(response_body.json()['errorKey'], 'UserNotFound')
```

and avoid the variable assignment.

Reply:
we chose to do this to keep the code consistent among the tests

Reply:
That's fine. There is at least one place that should conform to this then, e.g.

https://github.com/aau-giraf/web-api/blob/6e2c212fbff539eb2536d087813206273a1755e8/GirafIntegrationTest/tests/test_pictogram_controller.py#L234-L245

Also, just noticed, the snippet I linked could use the `params`-style for URL parameters I mentioned in another comment.